iogt_content_migration/management/commands/load_v1_db.py

```diff
@@ -541,6 +541,14 @@ def migrate_sections(self):
                             f'title: {translated_section.title}. URL: {translated_section.full_url}. '
                             f'Admin URL: {self.get_admin_url(translated_section.id)}.'
                         )
+                    self.post_migration_report_messages['disabled_larger_image_for_top_page_in_list_as_in_v1'].append(
+                        f'title: {translated_section.title}. URL: {translated_section.full_url}. '
+                        f'Admin URL: {self.get_admin_url(translated_section.id)}.'
+                    )
+                    self.post_migration_report_messages['disabled_comments_for_pages_which_has_other_settings'].append(
+                        f'title: {translated_section.title}. URL: {translated_section.full_url}. '
+                        f'Admin URL: {self.get_admin_url(translated_section.id)}. Original Setting: {commenting_status}.'
```

Comment:
This message should NOT appear if commenting was already disabled on the page. Only if was open/closed/inherited.
Same for all the other page types that have commenting.

Reply:
done